library/src/webapp/editor/ckextraplugins/pastebase64/plugin.js

```diff
@@ -0,0 +1,70 @@
+﻿(function () {

```

Comment:
I looks like this code was from here? https://github.com/javaha/ckeditor-pastebase64/blob/master/plugin.js

This would be better if this was packaged and included as webjar. https://www.webjars.org/

Reply:
@jonespm FerAlvarez95 signed the cla some months ago.
We don't know how to include this as webjar.

Reply:
It looks like it would need a tagged version to create a Bower Github webjar. So we'd have to fork it somewhere, create a tag and create it. Not sure where this fork would go. I also noticed that this has a PR against this repository which may need to be merged because of some issue with double paste.

Base64 for some images might go over the character size allowed for some fields in some places. When implementing something like this a long time ago in Samigo we'd run into those limits. I think it's by default only like 100K or 1MB in AntiSamy before it's blocked OOTB. Base64 encoding adds additional size to images, so that creates a new problem.

I also [saw this plugin](https://github.com/springalskey/ckeditor-plugin-paste-upload-image
) that is supposed to take uploaded images and upload them, but the biggest question is where do those uploads go? We have these special handlers to take care of audio recording uploads, I'm not sure if that would have needed to be used or we could just put them in regular results, which would depend on the user uploading where they'd go.

Reply:
This is an important regression for instructors. In 12 this functionality it was working!
We have this patch on our 20 production instance since 2 weeks ago.

If you want include this as webjar, file a new jira and change all the extra plugins including them as webjar.
Please merge. Thanks.

Reply:
We are going to close this one. No news since September.